### DISCUSSION TITLE
Cannot Download with VDH 8.2

### DISCUSSION THREAD

**Discussion by XangelMusic** [aclap-dev/video-downloadhelper#244] [Help and Q&A]
### Your question

So I believe this is part of the current ongoing issue with version 8.2 and figured I'd give my own details.
Below is the error I'm always getting and the details of my current setup

```
[in#1 @ 00000220ed35f740] Error opening input: Server returned 403 Forbidden (access denied)
Error opening input file https://vod.spwn.jp/spwn-vod/24022019-mgo/grp2/cam1_v1/cam1_v1_audio_320kbps_48k_.m3u8.
Error opening input files: Server returned 403 Forbidden (access denied)
```

I did download and install version 8.0.0.6 and I was able to download an mp4 video from the same website.

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 75052b43f0614124
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Josue Cerna",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
Lovely problem report except for one thing:  URLs.  If you want somebody to investigate your problem, you have to tell us what you are having trouble downloading.  Don't say "everything."  Give at least one URL that shows the symptoms you are experiencing.  When I say URL, I mean the URL of a web page on which one or more videos is/are hosted.  Don't try to save somebody's time by digging deep for a URL.  Just give the URL(s) of the original web page(s) where the video(s) reside(s).

**Reply by XangelMusic** [reply]
This is from SPWN, but I think you need a ticket to see the video.
https://virtual.spwn.jp/events/24022019-mgo/streaming

**Reply by GrampaWildWilly** [reply]
Right.  It prompts me for a log in.  In that case, I can't help you.  You might try the latest VDH beta.  @paulrouget released a new one last night.  To find it, click the link above labelled Wiki.  There's a Table of Contents on the right side of that page.  Click the entry that mentions beta releases.

**Reply by GrampaWildWilly** [reply]
If you're having trouble somewhere that doesn't require a login, that would be something there's a chance I could help you with.

**Reply by XangelMusic** [reply]
I just downloaded the beta (its say 8.2.2.0a1 beta in settings though), and it still gives me the same error as above.

Thanks anyway for your help.

**Reply by XangelMusic** [reply]
So I had a thought, but is there any way I can do some troubleshooting to assist regarding the issue?

**Comment by paulrouget** [comment]
@XangelMusic 

>  I was able to download an mp4 video from the same website.

It's a simple MP4 file? Does it help if in `settings > more > Video qualities` you enable "low quality hits" ?

**Reply by XangelMusic** [reply]
for which version of VDH? the latest stable version?

**Reply by XangelMusic** [reply]
@paulrouget 
Okay, so I updated to 8.2.1.1 stable, and did as you suggested. I was able to download only the video component (no audio), but this is not the same download result that I've been trying to get before which does still give me that error.

So in a way, this is an improvement, but there's no audio.

**Comment by XangelMusic** [comment]
@GrampaWildWilly @paulrouget So I'm thinking of closing this thread now, but I want to ask something first:

I'm able to download the video and audio of several streams on SPWN separately and then aggregate them into video files with their respective audios. I can't download a video with the audio by which VDH gives me that error above.

Is the purpose of VDH supposed to download video files with audio? Downloading the videos and audio is a workaround for me which can only be done per Paul's advice and allow "Low quality hits." Would you like me to further help you into finding a solution to this problem?
